timesketch/frontend-ng/src/components/LeftPanel/AnalyzerResults.vue

```diff
@@ -14,7 +14,11 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 <template>
-  <div>
+  <div v-if="iconOnly" class="pa-4" style="cursor: pointer" @click="$emit('toggleDrawer')">
```

Comment:
(not just here but also for all other icons)
Instead of just toggling the left panel when clicked on the icon, what do you think about also opening the clicked point. Currently this is two clicks, which could be one.

Example: Left panel is collapsed, when I now click the Saved Searches icon, the left panel will open and I have to click the saved searches section again to get the list shown.

Reply:
I thought about this as well but it didn't behave like I wanted it to. I can try again and see if I can get it to work nicely 👍🏼

Reply:
I was thinking of experimenting with it in a follow up PR, but I can give it a go in this one.

Reply:
Done!